File: 122562305-main CS50 course CODE/Lecture3/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct
    {
        string name;
        string number;
    }
    person;

int main(void)
{
    person people[2];

    people[0].name = "Carter";
    people[0].number = "795 229 666";

    people[1].name = "David";
    people[1].number = "604 781 882";

    string name = get_string("Name: ");

    for (int i = 0; i < 2; i++)
    {
        if (strcmp(people[i].name, name) == 0)
        {
            printf("Found: %s\n", people[i].number);
            return 0;
        }
    }
    printf("Not found\n"); 
    return 1;
}

File: 122562305-main CS50 course CODE/Lecture6/phonebook.py
import csv

file = open("phonebook.csv", "a")

name = input("Name: ")
number = input("Number: ")

writer = csv.writer(file)
writer.writerow([name, number])

file.close()


#-------------------------------

import csv

# Get name and number
name = input("Name: ")
number = input("Number: ")

# Open CSV file
with open("phonebook.csv", "a") as file:

    # Print to file
    writer = csv.DictWriter(file, fieldnames=["name", "number"])
    writer.writerow({"name": name, "number": number})

#---------------------------------------


people = {
    "Carter": "795 229 666",
    "David": "604 781 882"
}

name = input("Name: ")

if name in people:
    number = people[name]
    print(f"Number: {number}")